StPicoDstMaker/StPicoDstMaker.cxx

```diff
@@ -206,6 +217,20 @@ Int_t StPicoDstMaker::Init()
   return kStOK;
 }
 
+int StPicoDstMaker::setVtxModeAttr()
+{
+  if(TObject* vtxMode = (TObject*)(GetAttr()->FindObject("picovtxmode")))
```

Comment:
SAttr is a very informative name for a method ;-)
I didn't notice it. I will use.